### ISSUE TITLE
oa->å dipht. simpl. in actor nouns (

### ISSUE THREAD

**Issue by albbas** [giellalt/lang-smj#49]
This issue was created automatically with bugzilla2github
# Bugzilla Bug 193
Date: 2005-09-29T09:57:02+02:00
From: Thomas Omma &lt;<thomas.omma>&gt;
To:   Trond Trosterud &lt;<trond.trosterud>&gt;
CC:   borre.gaup, sjur.n.moshagen, trond.trosterud

Last updated: 2007-09-13T07:40:17+02:00

**Comment by albbas** [comment]
## Comment 575
Date: 2005-09-29 09:57:03 +0200
From: Thomas Omma &lt;<thomas.omma>&gt;

Concerns actor nouns from e-stem verbs.
In front of i there should be dipht simpl. This works with words like oade, which is in grade I, for 
example:
oadet+V+Actor+N+Sg+Com

**Comment by albbas** [comment]
## Comment 576
Date: 2005-09-29 10:23:43 +0200
From: Thomas Omma &lt;<thomas.omma>&gt;

Dipht simpl works when there are geminates too:

loagget+V+Actor+N+Pl+Gen
låggij

**Comment by albbas** [comment]
## Comment 578
Date: 2005-09-29 11:39:19 +0200
From: Thomas Omma &lt;<thomas.omma>&gt;

*** Bug #189 has been marked as a duplicate of this bug. ***

**Comment by albbas** [comment]
## Comment 580
Date: 2005-10-03 08:26:14 +0200
From: Thomas Omma &lt;<thomas.omma>&gt;

I understand Spiik this way: 
Grade I and II: diphtong simpl. before stem vowel -i-.
Grade III: no diphtong simpl. before stem vowel -i-.

This means that words boahtit->boahtte oadet->oade behave like they should do, but not loagget-
>loagge

**Comment by albbas** [comment]
## Comment 595
Date: 2005-10-11 12:51:46 +0200
From: Thomas Omma &lt;<thomas.omma>&gt;

A few testing examples after our G1-2-3 implementation. The should-be-forms in parenthesis.

boallo+N+Dim+N+Ess
boalutjin                       (bålutjin)

roavggo+N+Sg+Ill
råvgguj                        (roavgguj)


goarrot+V+Ind+Prs+Sg3
goarru                          (gårru)
goar'ru                         (RIGHT)

goarrot+V+Ind+Prs+Du1
goarrun                       (gårrun)
goar'run                      (RIGHT)

**Comment by albbas** [comment]
## Comment 648
Date: 2005-11-02 14:26:10 +0100
From: Thomas Omma &lt;<thomas.omma>&gt;

For our first testword it fails here:
toabmot+V+Pass+V+Inf
toamudallat

While for our second testword it does not fail with -dallat but in short passive:
oadot+V+Pass+V+Inf
oadut
ådudallat

This is not right either:
roavggo+N+Sg+Ill
råvgguj

roavggo+N+Dim+N+Ess
roavgutjin

**Comment by albbas** [comment]
## Comment 649
Date: 2005-11-03 15:09:30 +0100
From: Sjur Nørstebø Moshagen &lt;<sjur.n.moshagen>&gt;

Can you Thomas test all problem words, and check that they all go through now? And if so, you can also close the bug itself:-)

**Comment by albbas** [comment]
## Comment 653
Date: 2005-11-04 11:16:35 +0100
From: Thomas Omma &lt;<thomas.omma>&gt;

After some revisions to our "major revision" oa:å dipht. simpl. now works with stem-vowel -o - except for words with geminates=serie 2 in Spiik. Next step is therefore to deal with this.

We definitively made a major leap forwards with our revision. There are however some other things too that we have to deal with before we can close this bug:

oa:å dipht simpl. with other stem vowels than -o

æ:e dipht. simpl.

These will now be my prioritees.

**Comment by albbas** [comment]
## Comment 654
Date: 2005-11-05 16:00:35 +0100
From: Thomas Omma &lt;<thomas.omma>&gt;

It does not work with the central consonants gg:kk, dd:tt, bb:pp either: Series 6 in Spiik.

**Comment by albbas** [comment]
## Comment 655
Date: 2005-11-05 16:26:19 +0100
From: Thomas Omma &lt;<thomas.omma>&gt;

It does not work with the central consonants dtj:ttj, dts:tts: Series 5 in Spiik.
It does not work with the central consonants ks:vs etc. either: Series 7 in Spiik.

**Comment by albbas** [comment]
## Comment 659
Date: 2005-11-07 10:55:59 +0100
From: Thomas Omma &lt;<thomas.omma>&gt;

It does not work with the central consonants rssk or jsst either. These are series 3 in Spiik. Other series 3 work though

**Comment by albbas** [comment]
## Comment 663
Date: 2005-11-08 09:51:24 +0100
From: Thomas Omma &lt;<thomas.omma>&gt;

These do not work for S1:
ssjk:sjk
ssjm:sjm
ssjv:sjv

S3:
jsst:jst
lssk:lsk
msst:mst
nssk:nsk
nsst:nst
rssk:rsk
rssjt:rsjt
vsst:vst

**Comment by albbas** [comment]
## Comment 664
Date: 2005-11-08 10:05:56 +0100
From: Trond Trosterud &lt;<trond.trosterud>&gt;

rssk and jsst:
Here, I am not able to copy your failure. I did as follows: I copied the Alphabet, Sets and Definitions sections of twol-smj.txt over to a different file. Then I wrote a new Rules section, containing only the rule
"Test"
a:b <=> _ LowerG12 a ;
Then I opened the resulting file in twolc, compiled and tested:
prompt$ twolc -utf8
read-grammar test.txt
compile
lex-test
arska
arrska
etc.

I found a:b change for G2, but not for G3, as intended, also for these two consonant sequences. For these two, I thus predict that our rules should work. Also, all the G3 strings Thomas claims to be malfunctioning in his last posting work for me.

dtj, dts, ks, kt, ktj, kts, bb, dd, gg are all flawed, though, as they were not part of our initial setup. I will look into the issue.

**Comment by albbas** [comment]
## Comment 669
Date: 2005-11-08 11:27:37 +0100
From: Trond Trosterud &lt;<trond.trosterud>&gt;

Well, it seems my test rule wasn't good enough. I now ran it through the test bed in the twol file itself, and got 6 errors, among them our dear låvda. According to my test bed, the G2 of vdd (i.e. vd) is identified as G2 (i.e., as a member of the LowerG12 set).

This is a mystery to me.
